docs/userguide/datafiles.rst

```diff
@@ -172,7 +177,7 @@ file, nor require to be added by a revision control system plugin.
 
 .. note::
         If your glob patterns use paths, you *must* use a forward slash (``/``) as
-        the path separator, even if you are on Windows.  Setuptools automatically
+        the path separator, even if you are on Windows. ``Setuptools`` automatically
```

Comment:
As far as I remember in the docs we use capital Setuptools without monospaced font when referring to the project name and ``setuptools`` with monospaced font when referring to the package. Either one is fine, but ``Setuptools`` looks a bit off. I think it would be better if we use either the capital letter or the monospace font but not both.

```suggestion
        the path separator, even if you are on Windows. Setuptools automatically
```

Reply:
Thanks for the info, it looks like in this scenario we're more referring to the package (and therefore `setuptools` being more suitable)?

Reply:
Great, thank you!

Reply:
Thanks I would fold this